app/src/main/java/tech/ula/ui/FilesystemEditFragment.kt

```diff
@@ -155,14 +155,17 @@ class FilesystemEditFragment : Fragment() {
     }
 
     private fun filesystemParametersAreCorrect(): Boolean {
-        if (filesystem.name == "") input_filesystem_name.error = getString(R.string.error_filesystem_name)
-
+        val allCredentialsAreValid = true
         val validator = ValidationUtility()
-        var allCredentialsAreValid = false
         val username = filesystem.defaultUsername
         val password = filesystem.defaultPassword
         val vncPassword = filesystem.defaultVncPassword
 
+        if (filesystem.name.isEmpty()) {
+            input_filesystem_name.error = getString(R.string.error_filesystem_name)
+            return !allCredentialsAreValid
```

Comment:
I'd just return values from this function, rather than a variable. That way future readers don't have to spend any time tracking the logic.

Reply:
I removed the variable `allCredentialsAreValid` and used just a raw boolean.  I agree it'll make it easier to read